data/json/furniture_and_terrain/furniture-fungal.json

```
@@ -3,7 +3,7 @@
     "type": "furniture",
     "id": "f_flower_marloss",
     "name": "marloss flower",
-    "description": "This flower is like the other flowers taken by the mushrooms, but its bulb is colored a brilliant cyan color, and it emits an aroma both overwhelming and… delicious?",
+    "description": "This flower is like the other flowers taken by the mushrooms, but its bulb is colored a brilliant cyan color. It emits an aroma both overwhelming and strangely alluring.",
```

```suggestion
    "description": "This flower is like the other flowers taken by the mushrooms, but its bulb is colored a brilliant cyan color.  It emits an aroma both overwhelming and strangely alluring.",
```
This was the only remaining complaint in the error log that I could find.